src/SourceBuild/content/repo-projects/source-build-externals.proj

```diff
@@ -2,7 +2,10 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <BuildCommand>$(StandardSourceBuildCommand) $(StandardSourceBuildArgs)</BuildCommand>
+    <BuildCommandArgs>$(StandardSourceBuildArgs)</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) /p:UseInnerClone=true</BuildCommandArgs>
```

Comment:
Can you add a comment here as to why the inner clone is maintained in this case?

Reply:
> Can you add a comment here as to why the inner clone is maintained in this case?

Yes, will do - thanks!

Reply:
Fixed with https://github.com/dotnet/installer/pull/18153/commits/669b74ebe4d6880f5890aacdfed8029ccb778d6c